sdk/src/main/java/com/silanis/esl/sdk/internal/converter/SignerConverter.java

```diff
@@ -173,6 +177,10 @@ private Signer newSignerPlaceholderFromAPIRole() {
             signer.setLocked(true);
         }
 
+        if (apiRole.getData() != null && apiRole.getData().containsKey(Role.LOCAL_LANGUAGE_DATA_KEY)) {
+            signer.setLocalLanguage(apiRole.getData().get(Role.LOCAL_LANGUAGE_DATA_KEY).toString());
+        }
+
```

Comment:
As far as I know, we use the data from Role to set the local language. But do you need to do this? The SDK needs to handle the local language simply by passing data to the backend.